packages/decentraland-ecs/src/setupProxy.ts

```diff
@@ -61,12 +61,19 @@ const setupProxy = (dcl: any, app: express.Application) => {
   }
 
   createStaticRoutes(app, '/images/decentraland-connect/*', dclKernelImagesDecentralandConnect)
-  createStaticRoutes(app, '/@/artifacts/unity-renderer/*', dclUnityRenderer)
+  createStaticRoutes(app, '/@/artifacts/unity-renderer/*', dclUnityRenderer, (filePath) =>
+    filePath.replace(/.br+$/, '')
+  )
   createStaticRoutes(app, '/@/artifacts/loader/*', dclKernelLoaderPath)
   createStaticRoutes(app, '/default-profile/*', dclKernelDefaultProfilePath)
 }
 
-const createStaticRoutes = (app: express.Application, route: string, localFolder: string) => {
+const createStaticRoutes = (
+  app: express.Application,
+  route: string,
+  localFolder: string,
+  mapFile: ((filePath: string) => string) | null = null
```

Comment:
u can avoid that null default value making that arg optional with the `?` operator
```ts
mapFile?: (filePath: string) => string,
```

Reply:
smart, ty